bundles/org.openhab.binding.velbus/src/main/java/org/openhab/binding/velbus/internal/handler/VelbusBridgeHandler.java

```diff
@@ -68,6 +75,32 @@
 
     private boolean listenerStopped;
 
+    private final ChannelUID clockAlarm1Enabled = new ChannelUID(thing.getUID(), "bridgeClockAlarm",
+            "clockAlarm1Enabled");
```

Comment:
Rather work with channel IDs and define these IDs as constants in VelbusBindingConstants. This is the usual way to do.

Reply:
Changes done.